almond/config.json

```diff
@@ -1,17 +1,16 @@
 {
-  "name": "Almond",
-  "version": "1.1.2",
-  "slug": "almond",
-  "description": "The home server version of Almond",
+  "name": "Almond - Edge",
+  "version": "2.0.0-1",
+  "slug": "edge",
+  "description": "The open-source, privacy-preserving voice assistant",
   "url": "https://github.com/home-assistant/hassio-addons/blob/master/almond",
-  "arch": ["armv7", "armhf", "aarch64", "amd64", "i386"],
+  "arch": ["armv7", "aarch64", "amd64"],
```

Comment:
Any specific reason they are dropped?
I don't see any breaking change or information for the end-user about this in the changelog?

Reply:
They are dropped because they do not build correctly, due to missing packages or binary blobs for those architectures. For example, the version of node was updated to 12, and there are no builds of node 12 for armv6.

Reply:
> node 12 for armv6

There is. Also available on Alpine.

Reply:
On debian, we need use the nodesource repository for node 12, and that doesn't have node for armv6. I mean, you saw the build failure, right?

Reply:
Yep, but now learned about Snowboy 👍 
So I guess no choice at this point.